### PR TITLE
disable UWP on arm

### PR DESCRIPTION
**Pull request by wfurt**
it is failing all tests on all runs. 
It is clearly not ready.

### CODE REVIEW

File: eng/pipelines/windows.yml [hunk 1]
```diff
@@ -96,12 +96,6 @@ jobs:
               _framework: uap
               _helixQueues: $(uapNetfxQueues)
 
-            UAP_arm_Release:
```
Comment (safern) [inline eng/pipelines/windows.yml]: Can we at least keep building UAP without running tests? That will protect the build.

Comment (wfurt) [reply eng/pipelines/windows.yml]: I'm not sure how to do that? Remove only _helixQueues ???

Comment (safern) [reply eng/pipelines/windows.yml]: Nope, move it down to the Job called WindowsNoTest under the if statement IsOfficialBuild == true.

Comment (wfurt) [reply eng/pipelines/windows.yml]: done.

**Pull request merged**